### ISSUE TITLE
Door's sync

### ISSUE THREAD

**Issue by FRANkiller13** [qalle-fivem/esx_bankrobberies#8]
If the cop or someone else come to the bank he cant see the opened door 

Results:
![59755522_2102809833349885_4077562983553695744_n](https://user-images.githubusercontent.com/5957308/57150134-230d1680-6dd6-11e9-851b-1d4e17c7cfd0.jpg)

**Comment by wydox1SK** [comment]
Yes i also get this issue, very annoying. great script but this is a big problem!

**Comment by FRANkiller13** [comment]
@wydox1SK if you have problem with cashpile and alarm blip check my fix #7 the only fix i cant do is the door sync... i found something from s_doors and i dont where to start

**Comment by wydox1SK** [comment]
@FRANkiller13 Yea I've already applied your fixes mate so thank very much for them! I was thinking the door issue was an issue with onesync (i haven't tried it without onesync)

**Comment by FRANkiller13** [comment]
@wydox1SK If it works tell me to enable it too

**Comment by daviten74** [comment]
Tengo el mismo problema.
